src/images/Beer.svg

```diff
@@ -0,0 +1,3 @@
+</svg>
```

Comment:
Please run SVG images through SVGOMG to reduce the weight. You'll want to keep “Prefer viewBox to width/height” set to off for this.